sqlite/src/func.c

```diff
@@ -2700,6 +2700,7 @@ void sqlite3RegisterBuiltinFunctions(void){
     VFUNCTION(guid_str,          0, 0, 0, guidStrFunc      ),
     FUNCTION(guid,               1, 0, 0, guidFromStrFunc  ),
     FUNCTION(guid_str,           1, 0, 0, guidFromByteFunc ),
+    VFUNCTION(guid_func,         0, 0, 0, guidFunc         ),
```

Comment:
Why do we need this? We already have `VFUNCTION(guid, 0, 0, 0, guidFunc),` wouldn't that be sufficient?